spyder/widgets/mixins.py

```diff
@@ -130,18 +131,28 @@ def _update_stylesheet(self, widget):
                 }}'''.format(name, background, border)
             widget.setStyleSheet(css + extra_css)
 
-    def _format_text(self, title, text, color, ellide=False):
+    def _format_text(self, title, text, color, ellide=False,
+                     help_button=''):
         """
         Create HTML template for calltips and tooltips.
 
         This will display title and text as separate sections and add `...`
         if `ellide` is True and the text is too long.
         """
+        shortcut = 'Ctrl+I'  # FIXME:
+        help_link = ''
+        if help_button:
+            help_link = (
+            '<a href="{0}"><small>'
+            '<span style="float:right;color:white;">'
+            '(For more information, click here or press {1})'

```

Comment:
Before, the shortcut was formatted as a <kbd>keyboard button</kbd> (IIRC), which looked quite nice. Could you add ``<kbd>`` tags (or whatever else was used before)?